mysql-test/suite/galera/t/galera_truncate.test

```diff
@@ -103,3 +103,34 @@ TRUNCATE TABLE t1;
 INSERT INTO t1 (id, related_id) VALUES (1,1);
 
 DROP TABLE t1;
+
+#
+# TRUNCATE TABLE performance_schema.<table> should not be replicated
+#
+--disable_query_log
+--disable_result_log
+
+--connection node_2
+# We need to execute SELECT COUNT before geting the actual result to have it already in events_statements_summary_by_digest table 
```

Comment:
```suggestion
# We need to execute SELECT COUNT before getting the actual result to have it already in events_statements_summary_by_digest table 
```

Reply:
Fixed in 5.7 PR. 
Once https://github.com/percona/percona-xtradb-cluster/pull/1697 is approved, I will merge it to this PR according to GCA.